00-intro.md

```diff
@@ -69,8 +69,9 @@ Commands are terse (often only a couple of characters long),
 their names are frequently cryptic,
 and their output is lines of text rather than something visual like a graph.
 On the other hand,
-the shell allows us to combine existing tools in powerful ways with only a few keystrokes
-and to set up pipelines to handle large volumes of data automatically thus improving productivity and reproducibility.
+with only a few keystrokes shell allows us to combine existing tools into 
+powerful pipelines and handle large volumes of data automatically. This automation not only improves 
```

Comment:
I would suggest something a little more explicit about what is productive and reproducible:

"not only makes us more productive, but also improves the *reproducibility* of our workflows".

"Exactly the same workflow can be run multiple times" even by clicking exactly the same buttons every time. The stress should be on how easily this can be done on the shell, for instance:

... but also improves the *reproducibility* of our workflows" by allowing us to repeat them with just a few commands.